### DISCUSSION TITLE
Native mode [Windows] : Stay on top  option

### DISCUSSION THREAD

**Discussion by atticus-lv** [zauberzeug/nicegui#2955] [Ideas / Feature Requests]
### Description

hi there!
I notice that we need a stay on top feat on windows. I use os.system to run some command, after pack with pyinstaller, the cmd window will pop up then the nicegui window will be minimize(If not have a right foucs).

Video: (Left: pack, right: unpack)
<video src="https://github.com/zauberzeug/nicegui/assets/64118610/017b2118-941f-4b47-b61c-1e5adb531048" />

Maybe we should add a stay on top option when creating a native window
https://github.com/r0x0r/pywebview/pull/497/commits/d773580579dba6443847264d30554bf620061751

**Comment by falkoschindler** [comment]
Hi @atticus-lv,

Many webview parameters are already accessible from a NiceGUI app. You could try to set the `topmost` parameter like this:
```py
app.native.window_args['topmost'] = True
```

**Comment by atticus-lv** [comment]
Oh, that is amazing! thanks
